components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -17,2687 +17,4007 @@
+
+/* if Bluetooth Low Energy Feature Periodic Advertising Synchronization Transfer Support */ 
 #if (BLE_FEAT_PERIODIC_ADV_SYNC_TRANSFER == TRUE)
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COMPLETE_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COMPLETE_EVT event.
+     * This event is triggered when the periodic advertising receive enable operation completes.
      */
     struct ble_periodic_adv_recv_enable_cmpl_param {
-        esp_bt_status_t status;             /*!< Set periodic advertising receive enable status */
-    } period_adv_recv_enable;               /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COMPLETE_EVT */
+        esp_bt_status_t status;             /*!< Status of the periodic advertising receive enable operation. Indicates whether the operation was successful or if an error occurred. */
+    } period_adv_recv_enable;   /*!< ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COMPLETE_EVT */
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_SYNC_TRANS_COMPLETE_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_SYNC_TRANS_COMPLETE_EVT event.
+     * This event is triggered when the periodic advertising synchronization transfer operation completes.
      */
     struct ble_periodic_adv_sync_trans_cmpl_param {
-        esp_bt_status_t status;             /*!< Periodic advertising sync transfer status */
-        esp_bd_addr_t bda;                  /*!< The remote device address */
-    } period_adv_sync_trans;                /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_SYNC_TRANS_COMPLETE_EVT */
+        esp_bt_status_t status;             /*!< Status of the periodic advertising synchronization transfer operation. Indicates whether the transfer was successful or if an error occurred. */
+        esp_bd_addr_t bda;                  /*!< Address of the remote device involved in the synchronization transfer. */
+    } period_adv_sync_trans;    /*!< ESP_GAP_BLE_PERIODIC_ADV_SYNC_TRANS_COMPLETE_EVT */
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_SET_INFO_TRANS_COMPLETE_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_SET_INFO_TRANS_COMPLETE_EVT event.
+     * This event is triggered when the periodic advertising set information transfer operation completes.
      */
     struct ble_periodic_adv_set_info_trans_cmpl_param {
-        esp_bt_status_t status;             /*!< Periodic advertising set info transfer status */
-        esp_bd_addr_t bda;                  /*!< The remote device address */
-    } period_adv_set_info_trans;            /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_SET_INFO_TRANS_COMPLETE_EVT */
+        esp_bt_status_t status;             /*!< Status of the periodic advertising set information transfer operation. Indicates whether the transfer was successful or if an error occurred. */
+        esp_bd_addr_t bda;                  /*!< Address of the remote device involved in the information tran
```

Comment:
when vendor hci command complete, this ESP_GAP_BLE_VENDOR_CMD_COMPLETE_EVT comes.